.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,66 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
```

Comment:
I know it’s maybe a pain but for rendering bugs, if they are in wgpu or naga, they sometimes get fixed in point releases. In that case it is good to know which version of wgpu and naga are being used.

Reply:
Hmm. I think this is better tackled with a follow-up if they can reproduce it on Bevy but not in the `wgpu` examples.